packages/salesforcedx-vscode-core/package.nls.json

```diff
@@ -63,11 +65,10 @@
     "SFDX: Create and Set Up Project for ISV Debugging",
   "force_apex_log_get_text": "SFDX: Get Apex Debug Logs...",
 
-  "force_manifest_editor_show_text":
-    "SFDX: Show Package Manifest Editor (Experimental)",
-  "show_experimental_webviews_description":
-    "Specifies whether the commands for the experimental webviews are available to the user.",
-
   "retrieve_test_code_coverage_text":
-    "Specifies whether code coverage results are calculated and retrieved when you run Apex tests."
+    "Specifies whether code coverage results are calculated and retrieved when you run Apex tests.",
+
+  "change_set_based_tools.enabled_description":
+    "Specifies whether the commands for change-set-based development are available to the user.",
+  "force_manifest_editor_show_text": "SFDX: Show Package Manifest Editor"
```

Comment:
Although this is accurate, I fear that it might be confusing. Change-set-based development is being presented as an alternative to package-based development (i.e., second-generation packaging). Can we say something like, "SFDX: Show Manifest Editor for package.xml"?

Reply:
_This one probably doesn't need to be solved immediately_.

This one is tricky since it's not just limited to a file called package.xml. package.xml is the default one but the user can call it whatever she wants and can create multiple manifests. So putting package.xml in the command name might limit it from being used for differently named XML manifests.

Reply:
Ah, I see. Maybe @ntotten has thoughts on this.